apps/vue3/src/App.vue

```diff
@@ -3,104 +3,89 @@ import type { Container, Engine } from "tsparticles-engine";
 import { loadFull } from "tsparticles";
 import configs from "tsparticles-demo-configs";
 
-import HelloWorld from "./components/HelloWorld.vue";
-import TheWelcome from "./components/TheWelcome.vue";
-
 const options = configs.basic;
 
 const particlesInit = async (engine: Engine) => {
-    await loadFull(engine);
+  await loadFull(engine);
 
-    console.log(engine);
+  console.log(engine);
```

Comment:
This is unnecessary

Reply:
Oh sorry I misread it, the `console.log` already existed. I thought you added it. It can be removed since it's useless, if you want to remove it here is fine, otherwise I will remove it later